pkg/settings/setting.go

```diff
@@ -372,11 +373,25 @@ func GetEnvKey(key string) string {
 	return "CATTLE_" + strings.ToUpper(strings.Replace(key, "-", "_", -1))
 }
 
-func getMetadataConfig() string {
+type MetadataConfig struct {
+	URL                    string `json:"url"`
+	Path                   string `json:"path"`
+	RefreshIntervalMinutes int    `json:"refresh-interval-minutes"`
+}
```

Comment:
If we look at the function `parseURL` @ `pkg/controllers/management/kontainerdrivermetadata/url.go`, it supports the usage of a git repo (a url ending with `.git` + a branch). 
Even if we are not using it, we need to consider the possibility that there may be users who are using a git repo to host their clone of the KDM data. 


```suggestion
type MetadataConfig struct {
	URL                    string `json:"url"`
	Branch                string `json:"branch"`
	LocalPath                   string `json:"local-path"`
	RefreshIntervalMinutes int    `json:"refresh-interval-minutes"`
}
```